coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java

```diff
@@ -343,49 +343,35 @@ static Schema joinSchemas(@Nonnull Schema leftSchema, @Nonnull Schema rightSchem
     return combinedSchema;
   }
 
-  static Schema mergeUnionSchema(@Nonnull Schema leftSchema, @Nonnull Schema rightSchema, boolean strictMode) {
-    Preconditions.checkNotNull(leftSchema);
-    Preconditions.checkNotNull(rightSchema);
-
-    if (!leftSchema.toString(true).equals(rightSchema.toString(true))) {
```

Comment:
What's your thoughts of keeping this simple check? If they match, we do not need to do the merging logic. cc: @wmoustafa

Reply:
I kept it in the new version.